src/SupplierManager/Travely.SupplierManager.Repository/Abstractions/Entities/Accommodation/AccommodationEntity.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using TourEntities.Service.Accommodation;
+using TourEntities.Service.Common.Location;
+
+namespace Travely.SupplierManager.Repository.Entities
+{
+    public class AccommodationEntity : IEntity
+    {
+        [Key]
+        [Required]

```

Comment:
Why you are not using [AccommodationConfiguration](https://github.com/titancamp/travely/pull/252/files#diff-858609ee3f101ef0b1bc05225503bc63a9839de8be38495c3d137272b30816a4R7) class to describe the Entity state? because now the state of the Entity describes separately.
I see the same problem in other Entities too.